src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -124,6 +124,26 @@ TEST_CASE("generate load with unique accounts", "[loadgen]")
             },
             10 * Herder::EXP_LEDGER_TIMESPAN_SECONDS, false);
     }
+    SECTION("stop loadgen")
+    {
+        loadGen.generateLoad(GeneratedLoadConfig::createAccountsLoad(
+            /* nAccounts */ 10000,
+            /* txRate */ 1));
+        simulation->crankForAtLeast(std::chrono::seconds(10), false);
+        auto& acc = app.getMetrics().NewMeter({"loadgen", "account", "created"},
+                                              "account");
+        auto numAccounts = acc.count();
```

Comment:
Nit: Check that completed loadgen runs == 0 here to make sure we didn't just complete the accounts load before running stop.

Reply:
Also is 10 seconds of wall clock a little excessive here?

Reply:
>Nit: Check that completed loadgen runs == 0 here to make sure we didn't just complete the accounts load before running stop.

isn't it what the line below does? it goes from 0->1 on failed runs. 

> Also is 10 seconds of wall clock a little excessive here?
this is loopback mode, so not real time